src/main/app/src/Forms/EditForm.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+
+//{data} in parameters
+export default function FormDialog() {
```

Comment:
This is the other component I was talking about earlier - you can remove duplicate code in React by making a more generic component that is customizable via a handful of properties